src/main/kotlin/no/nav/familie/ef/iverksett/økonomi/IverksettMotOppdragTask.kt

```diff
@@ -43,7 +47,13 @@ class IverksettMotOppdragTask(private val iverksettingRepository: IverksettingRe
                                                                 forrigeTilkjentYtelse)
 
         tilstandRepository.oppdaterTilkjentYtelseForUtbetaling(behandlingId = behandlingId, utbetaling)
-        utbetaling.utbetalingsoppdrag?.let { oppdragClient.iverksettOppdrag(it) }
+        utbetaling.utbetalingsoppdrag?.let {
+            if (it.utbetalingsperiode.isNotEmpty()) {
+                oppdragClient.iverksettOppdrag(it)
+            } else {
+                log.warn("IverksettMotOppdragTask - iverksetter ikke noe mot oppdrag. Ingen utbetalingsperioder. behandlingId=$behandlingId")
```

Comment:
Loggern logger alltid klassen, så `IverksettMotOppdragTask` er unødvendig i meldingen
```suggestion
                log.warn("Iverksetter ikke noe mot oppdrag. Ingen utbetalingsperioder. behandlingId=$behandlingId")
```